apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -74,6 +74,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupInfered
               defaultValue={eventType.destinationCalendar || undefined}
               render={({ field: { onChange, value } }) => (
                 <DestinationCalendarSelector
+                  destinationCalendar={eventType.destinationCalendar}
```

Comment:
We didn't even pass in the calendar for the event to component 😬  So actually even if you selected something else, you would still see "default ()". That should be fixed now too.